Gemfile

```diff
@@ -1,4 +1,10 @@
 source 'https://rubygems.org'
+#
+# Add a Gemfile.local to locally bundle gems outside of version control
+local_gemfile = File.join(File.expand_path("..", __FILE__), "Gemfile.local")
+if File.readable?(local_gemfile)
+  eval_gemfile local_gemfile
+end
```

Comment:
I like using pry-byebug. This let's me do it without interfering with the project